keyboards/atreus62/keymaps/194h/keymap.c

```diff
@@ -0,0 +1,182 @@
+#include "atreus62.h"
```

Comment:
Lines 1-3 should be changed to `#include QMK_KEYBOARD_H`, which includes all of these files automatically.